### PR TITLE
trainController implemented using Thread

### PR DESCRIPTION
**Pull request by zolcsakgergo**

### CODE REVIEW

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java [hunk 1]
```diff
@@ -10,21 +10,20 @@ public class TrainControllerImpl implements TrainController {
 	private int step = 0;
 	private int referenceSpeed = 0;
 	private int speedLimit = 0;
-	private Timer time = new Timer();
+	private Thread t;
 
-	public TrainControllerImpl (){
```
Comment (zolcsakgergo) [inline train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java:15]: testcomment1

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java [hunk 2]
```diff
@@ -10,21 +10,20 @@ public class TrainControllerImpl implements TrainController {
...
-		
-	time.schedule(new TimerTask() {
```
Comment (zolcsakgergo) [inline train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java:17]: jobb a thread

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java (hunk 1 above)
Comment (Egi2018) [reply train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java:15]: Done

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java (hunk 2 above)
Comment (Egi2018) [reply train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java:17]: Done

**Pull request merged**